packages/grafana-ui/src/components/Monaco/CodeEditor.tsx

```diff
@@ -112,26 +117,30 @@ class UnthemedCodeEditor extends React.PureComponent<Props> {
 
       readOnly,
       lineNumbersMinChars: 4,
-      lineDecorationsWidth: 0,
+      lineDecorationsWidth: 1 * theme.spacing.gridSize,
       overviewRulerBorder: false,
       automaticLayout: true,
+      padding: {
+        top: 0.5 * theme.spacing.gridSize,
+        bottom: 0.5 * theme.spacing.gridSize,
+      },
     };
 
     if (!showLineNumbers) {
       options.glyphMargin = false;
       options.folding = false;
       options.lineNumbers = 'off';
-      options.lineDecorationsWidth = 5; // left margin when not showing line numbers
+      options.lineDecorationsWidth = 1 * theme.spacing.gridSize; // left margin when not showing line numbers
```

Comment:
I wanted to use the spacing function, but it outputs a `px` string, whereas this config must be a number.